### ISSUE TITLE
.csv import failed - invalid UTF-8 in field 0...

### ISSUE THREAD

**Issue by N-Demir** [quadratichq/quadratic#972]
Tried to paste the following csv into quadratic and received the error:

```
CSV parse error: record 0 (line 1, field: 0, byte: 0): invalid utf-8: invalid UTF-8 in field 0 near byte index 0
```
<img width="766" alt="Screenshot 2024-01-09 at 11 59 55 AM" src="https://github.com/quadratichq/quadratic/assets/23620644/149f8571-a12a-4ebc-8ddb-3ef027f4a5c9">

Maybe there's something weird about this csv file, but excel and google sheets had no problem opening it so I figured I should make an issue.

CSV File:
[Week 1.csv](https://github.com/quadratichq/quadratic/files/13878427/Week.1.csv)

**Comment by ddimaria** [comment]
There appear to be invalid characters in line 0 of the data rows:
![Screenshot 2024-01-09 at 1 04 13 PM](https://github.com/quadratichq/quadratic/assets/1536436/2dab6fe8-2103-4604-8e83-b7c60534c77b)

**Comment by N-Demir** [comment]
Thanks! That makes sense, and I figured there was something odd about it so it's my fault, but also wanted to bring attention to it because excel/gsheets didn't have a problem with it and maybe there's other situations in the future where users are surprised quadratic can't import a csv that the other ones can. Feel free to close!

**Comment by luke-quadratic** [comment]
closed, see #970 for future improvements to importing CSV files
